specs/specs.js

```diff
@@ -261,6 +261,44 @@ describe('Fingerprint2', function () {
       })
     })
 
+    describe('webgl shader precision format', function () {
+      it('checks webgl shader precision format loop', function (done) {
+
```

Comment:
Checking if all the corresponding entries are present. 
Couldn't think of a better approach for this spec. Please suggest if any.

Reply:
The problem with this kind of test is that it duplicates the code logic 1:1. Tests shouldn't try to be smart, as every little bit of smartness increases the chance of a bug in the tests.

My suggestion: Copy and paste the list of entries from the old fingerprint.js file into the tests, and compare them. It's find to have a hardcoded list of entries in the tests.

Reply:
Cool. Done.